install-sw.yml

```diff
@@ -72,9 +75,9 @@
     - name: rac-gi-install | defaults from common
       include_vars:
         dir: roles/common/defaults
-    - name: rac-gi-install | open firewall
-      include_role:
-        name: rac-lsnr-firewall
+  #  - name: rac-gi-install | open firewall
```

Comment:
No need for these commented-out lines, right?

Reply:
sorry, will clean up